docs/pages/programming/system_tables.md

```diff
@@ -120,30 +184,221 @@ quoted when used as an identifier.
 
     comdb2_keywords(name, reserved)
 
-* `name` - Name of the keyword.
-* `reserved` - 'Y' if the keyword is reserved, 'N' otherwise.
+* `name` - Name of the keyword
+* `reserved` - 'Y' if the keyword is reserved, 'N' otherwise
 
 ## comdb2_limits
 
 Describes all the hard limits in the database.
 
     comdb2_limits(name, description, value)
 
-* `name` - Name of the limit.
-* `description` - Description of the limit.
-* `value` - Value of the limit.
+* `name` - Name of the limit
+* `description` - Description of the limit
+* `value` - Value of the limit
 
-## comdb2_tunables
+## comdb2_locks
 
-Describes all the tunables in the database.
+Lists all active comdb2 locks.
 
-    comdb2_tunables(name, description, type, value, read_only)
+   comdb2_locks(thread, lockerid, mode, status, object, locktype, page)
+
+* `thread` - Thread Id of the owner thread
+* `lockerid` - Locker Id
+* `mode` - Lock mode (`DIRTY_READ`, `IREAD`, `IWR`, `IWRITE`, `NG`, `READ`,
+          `WRITE`, `WRITEADD`, `WRITEDEL`, `WAS_WRITE`, `WAIT`)
+* `status` - Status (`ABORT`, `ERROR`, `FREE`, `HELD`, `WAIT`, `PENDING`,
+             `EXPIRED`)
+* `object` - Locked object
+* `locktype` - Lock type (`PAGE`, `HANDLE`, `KEYHASH`, `ROWLOCK`, `MINMAX`,
+              `TABLELOCK`, `STRIPELOCK`, `LSN`, `ENV`)
+* `page` - Page number
+
+## comdb2_logical_operations
+
+Lists all logical operations
+
+    comdb2_logical_operations(commitlsn, opnum, operation, tablename, oldgenid, oldrecord, genid, record)
+
+* `commitlsn` - Log sequence number
+* `opnum` - Index of operation within a transaction
+* `operation` - Type of logical operation
+* `tablename` - Table name
+* `oldgenid` - Old record's generation Id
+* `oldrecord` - Old record
+* `genid` -  New record's generation Id
+* `record` - New record
+
+## comdb2_metrics
+
+Shows various operational and performance metrics.
+
+    comdb2_metrics(name, description, type, value, collection_type)
+
+* `name` - Name of the metrics
+* `description` - Description
+* `type` - Type of the metrics
+* `value` - Value
+* `collection_type` - Is value `cumulative` or `latest`?
```

Comment:
We can add a few sentences here to further explain what are cumulative and latest. For instance,

A cumulative metric is a cumulative sum over the time;
A latest metric is a instantaneous measurement.